sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePagedIterable.java

```diff
@@ -43,6 +50,34 @@ public ContinuablePagedIterable(ContinuablePagedFlux<C, T, P> pagedFlux, int bat
         super(pagedFlux);
         this.pagedFlux = pagedFlux;
         this.batchSize = batchSize;
+        this.defaultPageSize = 1;
```

Comment:
Why are we setting a default page size here? Shouldn't this be handled by the creator of this type?